index.html

```diff
@@ -25,5 +27,121 @@ <h3 class="dark-purple">My Logo</h3>
+                <p class="tag">css</p>
+                <p class="tag">javaScript</p>
+            </div>
+            <input class="alignment-start button" type="button" value="See Project">
+        </div>
+        <div class="card">
+            <img src="images/work-4-snapshot.svg" alt="Work 4 Image">
+            <h1>Multi-Post Stories</h1>
+            <div class="project-details">
+                <p>CANOPY</p>
+                <p class="dot">&#x2B24;</p>
+                <p class="project-details-color">Back End Dev</p>
+                <p class="dot">&#x2B24;</p>
+                <p class="project-details-color">2015</p>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags-section">
+                <p class="tag">html</p>
+                <p class="tag">css</p>
+                <p class="tag">javaScript</p>
+            </div>
+            <input class="alignment-start button" type="button" value="See Project" disabled>
+        </div>
+    </div>
+
+    <div class="about-section">
+        <p class="heading-color headline-text">About Myself</p>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="sub-heading">LET'S CONNECT</p>
+        <div>
+            <a href=""><img class="social-icon" src="images/twitter.svg" alt="twitter-icon"></a>
+            <a href="https://www.linkedin.com/in/awais-amjed/"><img class="social-icon" src="images/linkedin.svg" alt="linkedin-icon"></a>
+            <a href=""><img class="social-icon" src="images/medium.svg" alt="medium-icon"></a>
+            <a href="https://github.com/awais-amjed"><img class="social-icon" src="images/github.svg" alt="github-icon"></a>
+            <a href=""><img class="social-icon" src="images/peace.svg" alt="peace-icon"></a>
+        </div>
+        <input class="alignment-start button" type="button" value="Get My Resume">
+        <div class="blank-space"></div>
+        <div class="list-heading">
+            <p>Languages</p>
+            <i class="fas fa-chevron-down fa-xs"></i>
+        </div>
+        <ul>
+            <li>
+                <img src="images/javaScript.svg" alt="JS">
+                <p>Java Script</p>
+            </li>
+            <li>
+                <img src="images/html.svg" alt="JS">
+                <p>HTML</p>
+            </li>
+            <li>
+                <img src="images/css.svg" alt="JS">
+                <p>CSS</p>
+            </li>
+        </ul>
+        <div class="list-heading">
+            <p>Frameworks</p>
+            <i class="fas fa-chevron-right fa-xs"></i>
+        </div>
+        <div class="list-heading">
+            <p>Skills</p>
+            <i class="fas fa-chevron-right fa-xs"></i>
+        </div>
```

Comment:
- [ ] Please let's not forget to list our languages and skills.

Reply:
Aren't we supposed to do this in a later phase, when we fill the portfolio with our own information rather than the place-holder information?